scripts/python/lib/config.py

```diff
@@ -1504,6 +1526,67 @@ def yield_ntmpl_roles(self, node_template_index):
         for member in self.get_ntmpl_roles(node_template_index):
             yield member
 
+    def get_ntmpl_interfaces(self, node_template_index):
+        """Get node_templates interfaces
+        Args:
+            node_template_index (int): Node template index
+
+        Returns:
+            list of str: List of interface labels
```

Comment:
This says it's returning a list of interface labels, but the code looks to be returning more than that.

Reply:
Good catch, it is returning an interface dictionary.